content/docs/release-notes/archive/21-11-release-notes.md

```diff
@@ -0,0 +1,339 @@
+
+Right now we only have official support for Debian 20.04 LTS (Focal Fossa) as a pre-built binary distribution, but many Linux systems which meet the [hardware and software requirements](/docs/welcome-guide/requirements/) can be modified to successfully run Open 3D Engine. To get started building O3DE and your projects on Linux, check out the [Linux getting started documentation](docs/welcome-guide/setup/setup-from-github/building-linux/)
+
+And of course, you can [download the Debian package!](https://o3debinaries.org/download/linux.html)
+
+### Atom improvements
+
+Atom, the rendering library that powers Open 3D Engine, has received numerous improvements in this release, too.  Atom is now supported on Linux in a preview state alongside this release's support for the platform! In particular, AZSLc now compiles for Linux, null video devices no longer cause crashes, and [XCB](https://xcb.freedesktop.org/) connection support was added. The Vulkan rendering components of Atom were also all-up improved to help bring Linux support forward.
+
+More than just improvements and new platform support, Atom also now has improved debugging tools! First, the CPU profiler has been elevated to work with all of O3DE, allowing everyone to get more accurate profiling for their runtimes. We've also added support for [renderdoc]( https://renderdoc.org/) and provide a custom GPU visualizer so that you can perform advanced diagnostics on rendering pipelines.
+
+For content creators, Atom now has support for [TressFX 4.1]( https://github.com/GPUOpen-Effects/TressFX/) via the AtomTressFX Gem. With the AtomTressFX Gem, you have the power to create, model, and simulate realistic hair, fur, and other dense thin-volumed surfaces.
+
+### Editor improvements
+
+Open 3D Engine Editor 2111.1 has major improvements across the board in terms of performance and stability, like the rest of the engine, and content creation workflows are getting a lot of attention! Thanks to user feedback and sessions conducted with the O3DE UI/UX Special Interest Group, we collected a number of important adjustments our community wanted.
+
+Most importantly, Prefabs are now represented as a single object in the world, and you must use the **Edit Prefab** action on a Prefab in the Editor in order to edit the prefab itself. Prefab editing is available through **double-clicking** on an object in the Editor. While locked to the prefab edit mode, other instances of the prefab visible through the Editor viewport will receive real-time updates.
+
+And another change that we thought was so important that it deserves its own callout: This means is no more "sticky-select" in the Editor by default! You no longer have to double-click when deselecting an entity that you're editing. If you'd like to re-enable this feature, it can be changed in Editor Settings or with the `/Amazon/Preferences/Editor/StickySelect` registry setting.
```

Comment:
The first sentence is awkward. Either a key word was dropped after the ":" or it needs to be reworded. Unfortunately, I don't know what was meant here, so I can't offer a suggestion.